### PR TITLE
Turns miming into a liver trait

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

I didn't like how miming was stored in mind and caused bizarre checks for it, so i turned it into two separate organ traits - TRAIT_FRENCH_METABOLISM and TRAIT_BROKEN_VOW.
TRAIT_FRENCH_METABOLISM is always present, even if the vow is broken, and it grants you the vow of silence spell.
TRAIT_BROKEN_VOW only gets applied to the liver when your vow is broken. You'll still have the vow of silence action, but pretty much none of the benefits of french metabolism work.

Also, mime livers are always grayscaled.
image

I thought about making TRAIT_FRENCH_METABOLISM only affect how reagents are metabolized, but the thought of an entire mime's essence being stored in the liver is so fucking funny.

## Why It's Good For The Game

It felt really weird how mimes were the odd job out in regards to metabolism, so i fixed that.

## Changelog

:cl:
refactor: Mimery is now stored in the liver, you can steal someone's liver and implant them to get the vow of silence and the benefits that come from it.
/:cl:

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
i hit enter too early lol

**Comment by ATH1909** [comment]
won't this cause weirdness with stuff like the new genetics machine that replaces your organs with mutant ones

**Comment by ChungusGamer666** [comment]
> 

Only if it causes weirdness for every other job that has a metabolism trait (and a lot of these exist, so i doubt it)

**Comment by necromanceranne** [comment]
The liver trait stuff was created to handle unique drink interactions. What you want is skillchips.

**Comment by ChungusGamer666** [comment]
> 

fine i'll make it a skillchip thing even though miming being stored in the liver is funny

**Comment by ChungusGamer666** [comment]
actually no
i'm not turning this into a skillchip because i don't like skill chips and also making a skillchip additionally to being a liver trait would be maniacal and no one will bother removing both from a dead mime, since the vow of silence by itself does nothing worthwhile
additionally breaking the vow of silence should still make you lose mime metabolism, so the skillchip would directly interfere in how the liver works anyways

**Comment by ChungusGamer666** [comment]
> actually no i'm not turning this into a skillchip because i don't like skill chips and also making a skillchip additionally to a liver trait would be maniacal and no one will bother removing both from a dead mime, since the vow of silence by itself does nothing worthwhile additionally breaking the vow of silence should still make you lose mime metabolism, so the skillchip would directly interfere in how the liver works anyways

tldr unless maintainers want breaking the vow to not interfere with mime metabolism i won't turn this into a chip

**Comment by Farquaar** [comment]
I don't know, it doesn't make sense intuitively speaking. No player would expect that liver damage would make them less of a mime.

**Comment by VladinXXV** [comment]
The mime should not lose the power to be a mime over having their liver replaced. Mimery is a school of magic, and should remain tied to the mind as most if not all magic is. The benefits of drinking nothing? Sure, why not. But the power of mimery itself? Absolutely not.

**Comment by Time-Green** [comment]
I'm not convinced this is good for the game, and I tried to turn snails into livers. 

I think it's fine as is, it's a knowledge thing like krav maga, smoke, advanced mimery, languages etc. Miming is a mindset and is probably best as a mind thing, not a body thing. 

Skillchip is the best alternative if you really want it to be transferable

**Comment by ChungusGamer666** [comment]
yeah fuck this shit im not turning it into a skillchip

**Comment by Time-Green** [comment]
Understandable
